### ISSUE TITLE
[ENHANCEMENT 91563] Send zap command should check (and show) users balance

### ISSUE THREAD

**Issue by BenGWeeks** [BenGWeeks/Zapp.ie#4]
Zap amount is current set in Sats and allowed to be between 1 and 10,000 Sats. However, the user may not know their balance (and knowing their balance may affect what they would zap). We should add the allowance remaining to the adaptive card:

![image](https://github.com/user-attachments/assets/a0dc8216-5700-4431-b572-064454c3cb13)

**Comment by BenGWeeks** [comment]
Do we have this one logged @ediweeks?

**Comment by EdiWeeks** [comment]
Yes, i logged it. [Enhancement 91563](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/91563): Send zap command should check (and show) users balance
